apps/ui/src/pages/repo.tsx

```diff
@@ -287,6 +304,7 @@ function RepoWrapper({ children, id }) {
           flexGrow: 1,
           verticalAlign: "top",
           height: "100%",
+          ...(isSidebarOnLeftHand && { ml: open ? sidebar_width : 0 }),
```

Comment:
Any better way to adjust the Canvas position?

Reply:
This is the best way I know.